app/views/content/life-as-a-teacher/teaching-as-a-career/making-a-career-change-to-teaching.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Making a career change to teaching 
+title_paragraph: |-
+  <p>Peter shares how he switched careers to teaching, after more than 20 years working in finance.</p>
```

Comment:
```suggestion
  <p>Peter shares how he switched careers to teaching, after a long career in finance.</p>
```
If this is the para that appears on the page itself, I wonder whether we change it - the first line of the body content says `I spent 27 years....` so it seems a bit weird to have `more than 20 years` in the sentence before?